src/transforms/postcss.js

```diff
@@ -24,12 +26,16 @@ async function getConfig(asset) {
   }
 
   config = config || {};
-  config.plugins = loadPlugins(config.plugins, asset.name);
+  const postcssModulesConfig = Object.assign(get(config.plugins, 'postcss-modules') || {}, {
```

Comment:
Why not just `config.plugins['postcss-modules']`?

Reply:
type safety